From e8f41b58395a1d98def816cabcba91fa7af43212 Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Fri, 6 Apr 2012 13:56:27 +0800
Subject: [PATCH 297/442] ENGR00179064 Doc: Add stress test for power off and
 power on by RTC alarm

Add doc for pm test case

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 test/doc/ut_um.toc            |  1 +
 test/pm/doc/FSL-UT-PM-005.txt | 30 ++++++++++++++++++++++++++++++
 2 files changed, 31 insertions(+)
 create mode 100644 test/pm/doc/FSL-UT-PM-005.txt

diff --git a/test/doc/ut_um.toc b/test/doc/ut_um.toc
index 5c5f90b..73927bd 100644
--- a/test/doc/ut_um.toc
+++ b/test/doc/ut_um.toc
@@ -37,6 +37,7 @@ include::../pm/doc/FSL-UT-PM-001.txt[]
 include::../pm/doc/FSL-UT-PM-002.txt[]
 include::../pm/doc/FSL-UT-PM-003.txt[]
 include::../pm/doc/FSL-UT-PM-004.txt[]
+include::../pm/doc/FSL-UT-PM-005.txt[]
 
 PXP
 ---
diff --git a/test/pm/doc/FSL-UT-PM-005.txt b/test/pm/doc/FSL-UT-PM-005.txt
new file mode 100644
index 0000000..8eb48a9
--- /dev/null
+++ b/test/pm/doc/FSL-UT-PM-005.txt
@@ -0,0 +1,30 @@
+FSL-UT-PM-004-Stress-Power-Off-RTC-ALAM-Power-On
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+[cols=">s,6a",frame="topbot",options="header"]
+|====================================================================
+|Name | Description
+
+| Summary |
+Stree test for power off and power on boot by RTC alam
+
+| Automated |
+No
+
+| Kernel Config Option |
+
+| Test Procedure |
+.steps
+. copy below code into the begin of /etc/rc.d/init.d/udev
+
+ date
+ echo +10 > /sys/class/rtc/rtc0/wakealarm
+ poweroff -f
+
+| Expected Result |
+The board can power off and power on after 10s.
+Pass over night test.
+
+|=====================================================================
+
+<<<
-- 
1.8.0

